cmd/minishift/cmd/start_preflight.go

```diff
@@ -209,8 +209,20 @@ func checkKvmDriver() bool {
 	if err != nil {
 		return false
 	}
-	fmt.Printf(fmt.Sprintf("\n   Driver is available at %s ... ", path))
+	fi, _ := os.Stat(path)
+	// follow symlinks
+	if fi.Mode()&os.ModeSymlink != 0 {
+		path, err = os.Readlink(path)
+		if err != nil {
+			return false
+		}
+	}
+	fmt.Println(fmt.Sprintf("\n   Driver is available at %s ... ", path))
 
+	fmt.Printf("   Checking for execute bit ...")
```

Comment:
👍

Reply:
```diff
-fmt.Printf("   Checking for execute bit ...")
+fmt.Printf("   Checking for execute bit ... ")
```

Reply:
@gbraad @praveenkumar , any thought on 
`fmt.Printf("   Checking driver is executable ... ")` 
?